src/services/users.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../db');
+const { Expense } = require('../services/expenses');
+
+function normalize({ id, name }) {
```

Comment:
Function 'normalize' is not being used in this file. Consider removing it if it's not needed.